simpeg_drivers/components/factories/entity_factory.py

```diff
@@ -112,3 +129,44 @@ def _prune_from_indices(curve: Curve, cell_indices: np.ndarray):
         locations = curve.vertices[uni_ids, :]
         cells = np.arange(uni_ids.shape[0], dtype="uint32")[ids].reshape((-1, 2))
         return locations, cells
+
+    @staticmethod
+    def _validate_large_loop_cells(
+        transmitter: LargeLoopGroundFEMTransmitters | LargeLoopGroundTEMTransmitters,
+    ) -> np.ndarray:
+        """
+        Validate that the transmitter loops are counter-clockwise sorted and closed.
+        """
+        if transmitter.receivers.tx_id_property is None:
+            raise ValueError(
+                "Transmitter ID property required for LargeLoopGroundTEMReceivers"
+            )
+
+        tx_rx = transmitter.receivers.tx_id_property.values
+        tx_ids = transmitter.get_data("Transmitter ID")[0].values
```

Comment:
Outside the scope for this issue, but isn't this data required by the large loop ground em base class?  Shouldn't there be a property to access this?

Reply:
true